pvlib/tracking.py

```diff
@@ -570,3 +512,152 @@ def singleaxis(apparent_zenith, apparent_azimuth,
         out = {k: np.where(zen_gt_90, np.nan, v) for k, v in out.items()}
 
     return out
+
+
+def calc_axis_tilt(system_azimuth, system_zenith, axis_azimuth):
+    """
+    Calculate tracker axis tilt in the global reference frame when on a sloped
```

Comment:
Is "when on a sloped plane" necessary? It looks like this calculation works when `system_zenith` is 0.